build.gradle

```diff
@@ -321,6 +322,14 @@ project('segmentstore:storage:impl') {
 }
 
 project ('bindings') {
+    configurations.all {
```

Comment:
Can we move the resolution strategy to the global one in build.gradle, to enforce a common lib across all.
https://github.com/pravega/pravega/blob/729bc23b6370c2b9fb735e83ed496eec66a5991d/build.gradle#L109-L115

Reply:
Let me explain the rationale for the change a bit. 

We use Gson version 2.8.5 elsewhere, including in `:test:testcommon`. The `:bindings` project depends on both: 
* `:test:testcommon` sub-project, which brings in Gson `2.8.5` as transitive dependency. 
* HDFS and ECS libraries, which depend on/bring Gson `2.5` as transitive dependency. 

Since the two Gson versions are not compatible with each other, we run into version conflicts here. So, I had to force the use of the older version in the `:bindings` sub-project. But, we don't want to use the older version of Gson everywhere, that is why the config change is local to the `:bindings` project. If/when we upgrade HDFS and ECS dependencies, we might be able to remove this change. 

So, I think the suggested change might not be required. What do you say?

Reply:
Thanks @ravisharda , yeah we would end up downgrading gson version that is used by test common to avoid this. As discussed, this would change once we upgrade hdfs dependency...

Reply:
yeah, I've added a comment in the build file towards this end.